### PR TITLE
Added profile.

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1292"></a>

### CODE REVIEW

File: public/data/prajwalborkar.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/prajwalborkar",
+        "icon": "github"
+      },
+      {
+        "name": "Connect me on Linkedln",
+        "url": "https://www.linkedin.com/in/prajwal-borkar/",
+        "icon": "linkedin"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/prajwalborkar.json:18]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Shivansh-yadav13 — APPROVED** [review]
:+1:

**Review by emmalearnscode — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by kabir0x23 — APPROVED** [review]
Looks good 🌠

**Review by Panquesito7 — APPROVED** [review]

**Comment by PrajwalBorkar** [comment]
After merging I'm still unable to see my profile on the Website any issues?

**Comment by kumarsonsoff3** [comment]
> After merging I'm still unable to see my profile on the Website anyone knows about this?


**Merging does not automatically deploy, it will appear next time when the site gets deployed,** as I can see your profile in the main public/data directory

**Pull request merged**